README.md

```diff
@@ -13,15 +13,23 @@
 
 [API doc](https://williamdes.github.io/mariadb-mysql-kbs/Williamdes/MariaDBMySQLKBS.html)
 
-### In this respository you can find some data from knowledge bases
+
+### In this repository, you can find some data from knowledge bases.
 
 Raw extracted data : `/data/`
 
 Merged data : `/dist/`
 
-### Merged data
-- JSON format (raw, slim, ultraslim)
-- PHP format (ultraslim)
+# Merged Data Formats
+
+## JSON Format:
+- Raw
+- Slim
+- Ultraslim
+
+## PHP Format:
+- Ultraslim
```

Comment:
maybe add a link to the file in this repo
`[filename](./dist/filename)`

Reply:
ok